AutoCollection/HttpDependencyParser.ts

```diff
@@ -46,7 +43,7 @@ class HttpDependencyParser extends RequestParser {
      * Gets a dependency data contract object for a completed ClientRequest.
      */
     public getDependencyTelemetry(baseTelemetry?: Contracts.Telemetry, dependencyId?: string): Contracts.DependencyTelemetry {
-        let urlObject = url.parse(this.url);
+        let urlObject = new url.URL(this.url);
```

Comment:
When `this.url` is used, it is the full url (including protocol) so ignore `base`.